test/f90_correct/src/check_mod.F90

```diff
@@ -933,6 +937,95 @@ subroutine checkc8(reslt, expct, np, atoler, rtoler, ulptoler, ieee)
       endif
       return
     end subroutine checkc8
+    ! complex*32
```

Comment:
please move this comment inside of the `#ifdef` block

Reply:
OK, already updated.